### DISCUSSION TITLE
Is it work correctly on rtl pages?

### DISCUSSION THREAD

**Discussion by mhmdshz** [idiotWu/smooth-scrollbar#466] [Q&A]
When I was working with 
'smooth-scrollbar-react" I found out that it uses "smooth-scrollbar" as it's core, but "smooth-scrollbar-react"  doesn't work correctly on rtl pages

**Comment by dolphin-wood** [comment]
This plugin does not support rtl at this moment, but I am considering adding it in v9.

**Reply by mhmdshz** [reply]
I used this plugin for a bog project , almost 70 pages . If you would I could help you to improve it

**Reply by dolphin-wood** [reply]
Thank you! It would be great if you could help as I know little about rtl layouts. 🤗

**Reply by mhmdshz** [reply]
if page is rtl:
1-reverse position-y right or left  and   2-translate3d horizontall multiply by  (-) and ,..
you can get  rtl value from document.getElementsByTagName("html")[0].getAttribute("dir");

**Reply by mhmdshz** [reply]
https://github.com/idiotWu/smooth-scrollbar/issues/369
